tests/vmcheck/test-layering-non-root-caps.sh

```diff
@@ -107,7 +107,19 @@ check_group() {
 
 check_fcap() {
   local fcap=$(vm_cmd getcap $1)
-  local fcap=${fcap#* = } # trim filename
+  local fcap=${fcap#* = } # trim filename for pre-2.48 libcap: /usr/bin/foo = cap_net_raw+ep
+  fcap=${fcap#* } # And from the new 2.48+ libcap: /usr/bin/foo cap_net_raw=ep
+  if test -z "$2"; then
+    if test -n "$fcap"; then
+      assert_not_reached "expected no fcaps but found $fcap"
+    fi
+    return
+  fi
+  # Replace '+' with '='; a libcap change https://bodhi.fedoraproject.org/updates/FEDORA-2021-eeff266a64
```

Comment:
That should be https://bodhi.fedoraproject.org/updates/FEDORA-2021-570cc05441, no?

Reply:
Have a fix for that queued up in https://github.com/coreos/rpm-ostree/pull/2583 - let's get this in and unblock CI and avoid another respin?